examples/bq-email-exports/source_2/main.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Cloud Function for exporting BigQuery results from an anonymous table to GCS.
+Triggered after BigQuery query is complete.
+"""
+
+import base64
+import json
+import logging
+
+from google.cloud import bigquery
+
+
+def main(event, context):
+    """Entrypoint for Cloud Function"""
+
+    # Set variables
+    bucket_name = "bq-email-exports"
+    object_name = "bq_email_results.json"
```

Comment:
should this be `bq_email_results-*.json` just in case the results > 1GB?

Reply:
In the case the results are > 1 GB, the user would have to refractor the source code to extract multiple BQ tables into GCS and then use GCS compose to concatenate the results into one file before emailing. Adding `bq_email_results-*.json` as the default name may imply that this solution does support results > 1GB. Ultimately, I'll be making this name configurable in an env variable though.